pkg/objstore/s3/s3.go

```diff
@@ -32,6 +32,10 @@ import (
 // DirDelim is the delimiter used to model a directory structure in an object store bucket.
 const DirDelim = "/"
 
+// Minimum file size after which an HTTP multipart request should be used to upload objects to storage.
+// Set to 128 MiB as in the minio client.
+const minPartSize = 1024 * 1024 * 128
```

Comment:
```suggestion
const defaultMinPartSize = 1024 * 1024 * 128
```
.. might be more descriptive here, but not a blocker (:

Reply:
fixed